HousingSearchApi/V1/Boundary/Responses/GetAccountListResponse.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections.Generic;
+using Hackney.Shared.HousingSearch.Domain.Accounts;
+
+namespace HousingSearchApi.V1.Boundary.Responses
+{
+    public class GetAccountListResponse
```

Comment:
Suggest to follow the `public static GetAccountListResponse Create()` with private constructor/setters...